ni/github.py

```diff
@@ -38,6 +38,8 @@
 and a Python core developer will remove the `CLA not signed` label
 to make the bot check again.
 
+(How to check if CLA has been received)[https://devguide.python.org/committing.html#checking-if-the-cla-has-been-received].
```

Comment:
I think I would word this as "(You can [check yourself](https://devguide.python.org/committing.html#checking-if-the-cla-has-been-received) to see if the CLA has been received.)"

And your Markdown syntax is reversed. 😉

Reply:
😅 Thanks, fixed.